### DISCUSSION TITLE
Can I join #TEAM

### DISCUSSION THREAD

**Discussion by DESTROYER3264** [ROM-builders/temporary#15378] [Q&A]
Hello Sir,

I was previously a push-user and my github acc got suspended so I am applying again for push-user in ROM-Builders team

My github Profile: https://github.com/DESTROYER3264
My Telegram: https://t.me/D34DPUL32

My Successful Build:
https://cirrus-ci.com/build/5072148205666304
https://cirrus-ci.com/build/6613568347439104

My Device Tree:
https://github.com/DESTROYER3264/device_motorola_liber

Thanks

**Comment by Apon77** [comment]
why it got suspended? also your github and telegram username does not match

**Reply by DESTROYER3264** [reply]
Not sure about why my github acc got suspended, I haven't done anything shady in previous account.
I have changed my telegram username now to https://t.me/DESTROYER3264

**Reply by Apon77** [reply]
what is your previous branches/github username? there is 0 clue that you were push user previously.

**Reply by DESTROYER3264** [reply]
liber-PixelOS-Pixelish-twelve-D34DPUL
liber-Project-Awaken-12.1-D34DPUL
these are my previous branches

**Reply by Apon77** [reply]
oh, but your current profile does not even says its real, also no indication that you know #learn things.

**Reply by DESTROYER3264** [reply]
ok No problem if you don't want to add me up i'll remain a PR user. Thanks for the services.
